package.json

```diff
@@ -22,12 +22,13 @@
     "typescript": "^3.3.1"
   },
   "scripts": {
-    "postinstall": "lerna bootstrap",
+    "postinstall": "lerna bootstrap && npm run -s update-greenkeeper-json",
```

Comment:
This is not really solving the problem, is it? If the person does not run install after adding a new package, or perhaps revert changes made to greepkeeper config, then we won't know the pull request is not complete.

What I would like to see instead:
- when `update-greenkeeper-json.js` is run with no arguments (or perhaps with a special new arg), and the config file requires an update, then the tool exits with non-zero status code
- add a new entry to `.travisyml` to run consistency check, see commit-lint and code-lint for inspiration
- add a new code block to `bin/travis.sh` implementing the new check by running `update-greenkeeper-json.js`

I am ok to defer this changes to a follow-up pull request if you prefer.

Reply:
Let's get greenkeeper to do the job first as it awaits this PR to be landed :-).